pkg/controllers/openshift-route-controller-manager.go

```diff
@@ -24,48 +24,48 @@ import (
 
 	"k8s.io/klog/v2"
 
-	openshift_controller_manager "github.com/openshift/openshift-controller-manager/pkg/cmd/openshift-controller-manager"
+	route_controller_manager "github.com/openshift/route-controller-manager/pkg/cmd/route-controller-manager"
 
 	"github.com/openshift/microshift/pkg/assets"
 	"github.com/openshift/microshift/pkg/config"
 	"github.com/openshift/microshift/pkg/util"
 	"github.com/openshift/microshift/pkg/util/cryptomaterial"
 )
 
-type OCPControllerManager struct {
+type OCPRouteControllerManager struct {
 	kubeconfig     string
 	ConfigFilePath string
 	Output         io.Writer
 }
 
 const (
-	// OCPControllerManager component name
-	componentOCM = "openshift-controller-manager"
+	// OCPRouteControllerManager component name
+	componentRCM = "route-controller-manager"
 )
 
-func NewOpenShiftControllerManager(cfg *config.MicroshiftConfig) *OCPControllerManager {
-	s := &OCPControllerManager{}
+func NewRouteControllerManager(cfg *config.MicroshiftConfig) *OCPRouteControllerManager {
+	s := &OCPRouteControllerManager{}
 	s.configure(cfg)
 	return s
 }
 
-func (s *OCPControllerManager) Name() string { return componentOCM }
-func (s *OCPControllerManager) Dependencies() []string {
+func (s *OCPRouteControllerManager) Name() string { return componentRCM }
+func (s *OCPRouteControllerManager) Dependencies() []string {
 	return []string{"kube-apiserver", "openshift-crd-manager"}
 }
 
-func (s *OCPControllerManager) configure(cfg *config.MicroshiftConfig) {
+func (s *OCPRouteControllerManager) configure(cfg *config.MicroshiftConfig) {
 	if err := s.writeConfig(cfg); err != nil {
-		klog.Fatalf("Failed to write openshift-controller-manager config %v", err)
+		klog.Fatalf("Failed to write route-controller-manager config %v", err)
 	}
 
-	var configFilePath = cfg.DataDir + "/resources/openshift-controller-manager/config/config.yaml"
+	var configFilePath = cfg.DataDir + "/resources/route-controller-manager/config/config.yaml"
 	s.kubeconfig = filepath.Join(cfg.DataDir, "resources", "kubeadmin", "kubeconfig")
 	s.ConfigFilePath = configFilePath
 	s.Output = os.Stdout
 }
 
-func (s *OCPControllerManager) writeConfig(cfg *config.MicroshiftConfig) error {
+func (s *OCPRouteControllerManager) writeConfig(cfg *config.MicroshiftConfig) error {
```

Comment:
Another comment that came to mind. The idea is to standardize the way we configure the embeded components to avoid writing config files in disk. 

Could you please update this PR to use structs instead like the CPC?

https://github.com/openshift/microshift/blob/main/pkg/controllers/cluster-policy-controller.go#L53

Reply:
yes, planning to add it, although it might not be that pretty as CPC since we are not using the controller builder (yet)